aviary/docs/user_guide/aviary_commands.ipynb

```diff
@@ -172,6 +172,49 @@
    "source": [
     "!aviary fortran_to_aviary -h"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "(aviary-hanger-command)=\n",
+    "### aviary hanger\n",
+    "\n",
+    "The `aviary hanger` command will download files and folders from the Aviary hanger.\n",
+    "This is useful for users that have pip installed Aviary, but want to experiment with the included examples.\n",
+    "\n",
+    "The only required input is the name of an input deck.\n",
+    "This can be specified as the name of the file, the path from aviary/models, the name of a folder in aviary/models. Multiple files and folders can be downloaded at once.\n",
```

Comment:
Please add a link to the GitHub repo at aviary/models so people can see what's available